### PR TITLE
(fix) db init script to include vizualization

### PR DESCRIPTION
**Pull request by mark-antony1**

### CODE REVIEW

File: db_init_script.py [hunk 1]
```diff
@@ -4,6 +4,9 @@
 from merkato.utils.database_utils import no_merkatos_table_exists, create_merkatos_table, insert_merkato, get_all_merkatos, get_exchange
 from merkato.utils import generate_complete_merkato_configs
```
Comment (nasaWelder) [inline db_init_script.py:5]: Add diagnostics to import

**Review by nasaWelder — CHANGES_REQUESTED** [review]

File: db_init_script.py (hunk 1 above)
Comment (mark-antony1) [reply db_init_script.py:5]: agreed

**Review by needmoney90 — APPROVED** [review]

**Comment by nasaWelder** [comment]
Note visualize order book is blocking. Will depracate when I fix GUI for new formats @livinginformation

**Review by nasaWelder — APPROVED** [review]

**Comment by needmoney90** [comment]
Merged

**Pull request merged**